101-aci-linuxcontainer-public-ip/azuredeploy.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {    
+  },
+  "variables": {
+    "name": "acilinuxpublicipcontainergroup",
+    "image": "microsoft/aci-helloworld",   
+    "port": "80",
```

Comment:
I was following the best practices doc, which indicates that variables should be preferred over params except for things that must be unique or are secret:

https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#best-practices

I can change it to params if you think that's better.